src/softerllama/modeling_softerllama.py

```diff
@@ -195,7 +195,7 @@ def forward(
         # upcast attention to fp32
         # softermax overriding original attention softmax
         attn_weights = softermax(attn_weights, self.n_bias, dim=-1).type(dtype=torch.float32).to(query_states.dtype)
-        attn_weights = clip_logits(attn_weights, self.n_clip, dim=-1)
+        attn_weights = clip_logits(attn_weights, self.n_clip)
```

Comment:
@photomz double check clip_logits was throwing an error since `dim=-1` doesn't exist? Can you check that deleting it is fine and I didn't have to transpose something.